modules/library/src/main/scala/zio/elasticsearch/Boost.scala

```diff
@@ -16,8 +16,17 @@
 
 package zio.elasticsearch
 
-import zio.elasticsearch.ElasticQuery.{ElasticPrimitive, MatchAllQuery, TermQuery, WildcardQuery}
-import zio.elasticsearch.ElasticQueryType.{MatchAll, Term, Wildcard}
+import zio.elasticsearch.ElasticQuery.{
+  BoolQuery,
+  ElasticPrimitive,
+  LowerBound,
+  MatchAllQuery,
+  RangeQuery,
+  TermQuery,
+  UpperBound,
+  WildcardQuery
+}
```

Comment:
Can you import `zio.elasticsearch.ElasticQuery._`?

Reply:
Changes done !